docs/DeveloperGuide.adoc

```diff
@@ -729,9 +729,87 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`* * *` |forgetful user |set reminders |do not forget what I need to do
+
+|`* * *` |user |export the list of people in my address book |store it on the computer
+
+|`* * *` |user |set events and send the event invites to any person in my contact list |sync my calendars with others' on our common events
+
+|`* * *` |user |attach personal notes to people in my contact list |take notes about the person for my own reference
+
+|`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else
+seeing them by accident
+
+|`* *` |user |send mass messages to a group of contacts |plan for meetings, conferences or gatherings with ease
+
+|`* *` |user |drag and drop contacts in or out of different groups |not need to key in complex command lines
+
+|`* *` |user |view groups of contacts in a more visual display |see who are also in the group
+
+|`* *` |user |view groups of contacts in a more visual display |look for a contact in the group easily
+
+|`* *` |user |commands to be in the button form |avoid memorising the commands and select them more easily
+
+|`* *` |user |have meta-data for my contact list |sort them according to different criteria
+
+|`* *` |user |the application not to take up too much space on my device |have space for other applications
+
+|`* *` |user |the application should suggest certain fields for autocomplete |do not have to type them
+
+|`* *` |user |easily send emails to my contacts |conveniently contact my friends
+
+|`* *` |user |update my location |let my contacts know my current location
+
+|`* *` |user |update my status |contacts will know whether I’m busy
+
+|`* *` |user |view live stories of my friends |stay updated on what is going on with my friends
+
+|`* *` |user |start a live chat with my friends |get an instant response from them
+
+|`* *` |user |add a profile picture |let others know how I look like
+
+|`* *` |user |change the colour scheme |make it personalized
+
+|`* *` |user |choose not to cache the password after every login |ensure that the application is secure
+
+|`* *` |user |access the applications quickly |be efficient with  what I do
+
+|`* *` |user |avoid switching between applications for different functionality |avoid troublesome operations for me
+
+|`* *` |user |add profiles for my friends |store their photos as well
 
 |`*` |user with many persons in the address book |sort persons by name |locate a person easily
+
+|`*` |user |have a collage of a person's profile from various social media platforms |snapshot of the person's information
+from various accounts.
+
+|`*` |user |subscribe to different channels |get more information on the outside world
+
+|`*` |user |play music in the background |feel relaxed while using the application
+
+|`*` | elderly user |application to read the contacts out loud |avoid straining my eyes reading the words
 |=======================================================================
 
```

Comment:
You have a wide range of user stories. Maybe you can slowly narrow it down more for your project scope.

Reply:
Ok, but do we need to remove it out of the User Stories?

Reply:
You can keep it here for now, but the priorities should be modified.